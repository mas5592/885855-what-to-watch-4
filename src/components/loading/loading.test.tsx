import * as React from 'react';
import * as renderer from "react-test-renderer";
import Loading from './loading';

describe(`Loader`, () => {
  it(`Should render loading correctly`, () => {
    const tree = renderer
      .create(
          <Loading />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
