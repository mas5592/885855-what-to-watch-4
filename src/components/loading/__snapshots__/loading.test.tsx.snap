// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Should render loading correctly 1`] = `
<div
  className="user-page"
>
  <div
    className="sign-in user-page__content"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "center",
        "minHeight": "100vh",
        "width": "100%",
      }
    }
  >
    <div
      dangerouslySetInnerHTML={
        Object {
          "__html": "<svg class=\\"icon-loading\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 32 32\\">
  <path transform=\\"translate(-8 0)\\" d=\\"M4 12 A4 4 0 0 0 4 20 A4 4 0 0 0 4 12\\"> 
    <animateTransform attributeName=\\"transform\\" type=\\"translate\\" values=\\"-8 0; 2 0; 2 0;\\" dur=\\"0.8s\\" repeatCount=\\"indefinite\\" begin=\\"0\\" keytimes=\\"0;.25;1\\" keySplines=\\"0.2 0.2 0.4 0.8;0.2 0.6 0.4 0.8\\" calcMode=\\"spline\\"  />
  </path>
  <path transform=\\"translate(2 0)\\" d=\\"M4 12 A4 4 0 0 0 4 20 A4 4 0 0 0 4 12\\"> 
    <animateTransform attributeName=\\"transform\\" type=\\"translate\\" values=\\"2 0; 12 0; 12 0;\\" dur=\\"0.8s\\" repeatCount=\\"indefinite\\" begin=\\"0\\" keytimes=\\"0;.35;1\\" keySplines=\\"0.2 0.2 0.4 0.8;0.2 0.6 0.4 0.8\\" calcMode=\\"spline\\"  />
  </path>
  <path transform=\\"translate(12 0)\\" d=\\"M4 12 A4 4 0 0 0 4 20 A4 4 0 0 0 4 12\\"> 
    <animateTransform attributeName=\\"transform\\" type=\\"translate\\" values=\\"12 0; 22 0; 22 0;\\" dur=\\"0.8s\\" repeatCount=\\"indefinite\\" begin=\\"0\\" keytimes=\\"0;.45;1\\" keySplines=\\"0.2 0.2 0.4 0.8;0.2 0.6 0.4 0.8\\" calcMode=\\"spline\\"  />
  </path>
  <path transform=\\"translate(24 0)\\" d=\\"M4 12 A4 4 0 0 0 4 20 A4 4 0 0 0 4 12\\"> 
    <animateTransform attributeName=\\"transform\\" type=\\"translate\\" values=\\"22 0; 32 0; 32 0;\\" dur=\\"0.8s\\" repeatCount=\\"indefinite\\" begin=\\"0\\" keytimes=\\"0;.55;1\\" keySplines=\\"0.2 0.2 0.4 0.8;0.2 0.6 0.4 0.8\\" calcMode=\\"spline\\"  />
  </path>
</svg>
",
        }
      }
      style={
        Object {
          "fill": "#372824",
          "height": 64,
          "width": 64,
        }
      }
    />
  </div>
</div>
`;
